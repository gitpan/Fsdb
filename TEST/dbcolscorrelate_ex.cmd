prog='dbcolscorrelate'
args='--sample test1 test2'
cmp='diff -cb '
