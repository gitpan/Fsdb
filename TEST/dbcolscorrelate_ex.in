#h name id test1 test2
a 1 80 81
b 2 70 71
c 3 65 66
d 4 90 91
e 5 70 71
f 6 90 91
