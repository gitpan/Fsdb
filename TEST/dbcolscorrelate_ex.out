#fsdb correlation
0.83329
#   | dbcolscorrelate --sample test1 test2
